### PR TITLE
Update Endpoints samples:

### PR DESCRIPTION
**Pull request by sepehre**
1. Change default service names to the new cloud.goog format.
2. Change app.yaml beta_settings to endpoints_api_service.

### CODE REVIEW

**Comment by sepehre** [comment]
Please do not merge until documentation updated have been coordinated.

File: endpoints/getting-started/app.yaml [hunk 1]
```diff
@@ -5,8 +5,6 @@ entrypoint: gunicorn -b :$PORT main:app
 runtime_config:
   python_version: 3
 
-beta_settings:
-  # Enable Google Cloud Endpoints API management.
-  use_endpoints_api_management: true
-  # Specify the Open API specification.
-  endpoints_swagger_spec_file: openapi.yaml
+endpoints_api_service:
+  name: ENDPOINTS SERVICE-NAME # To be replaced.
```
Comment (jonparrott) [inline endpoints/getting-started/app.yaml]: To be replaced but by what? Can you update your comment to be more specific?

**Comment by jonparrott** [comment]
@sepehre once approved, let us know when you want to merge. We'll remove the `DO NOT MERGE` label.

File: endpoints/getting-started/app.yaml [hunk 2]
```diff
@@ -5,8 +5,6 @@ entrypoint: gunicorn -b :$PORT main:app
 runtime_config:
   python_version: 3
 
-beta_settings:
-  # Enable Google Cloud Endpoints API management.
-  use_endpoints_api_management: true
-  # Specify the Open API specification.
-  endpoints_swagger_spec_file: openapi.yaml
+endpoints_api_service:
+  name: ENDPOINTS SERVICE-NAME # To be replaced by the output of 'gcloud service-management deploy openapi.yaml' command.
```
Comment (sepehre) [inline endpoints/getting-started/app.yaml]: Good idea, thanks.

**Comment by sepehre** [comment]
Please merge.

**Pull request merged**